### ISSUE TITLE
Crash on client connection

### ISSUE THREAD

**Issue by LemADEC** [ToroCraft/ToroHealth#6]
Client crashes randomly when connecting to server

```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  LoadingPlugin (Quark-r1.0-56.jar)
  ShetiPhian-ASM (shetiphiancore-1.10.0-3.3.1.jar)
  LoadingPlugin (sampler-1.66.jar)
  llibrary (llibrary-1.7.0-1.10.2.jar)
  AppleCore (AppleCore-mc1.10.2-2.0.1.jar)
  HFTransformer (Harvest-Festival-1.10.2-0.5.3.jar)
  LoadingPlugin (ResourceLoader-MC1.9.4-1.5.1.jar)
  FMLPlugin (InventoryTweaks-1.61-58.jar)
  SCFMLCorePlugin (StalkerCreepers-1.10.2.jar)
  BCModPlugin (zz-backpacks 1.10.2 - 3.1.2.jar)
  BiomeTweakerCore (BiomeTweaker-1.10.2-2.1.225.jar)
  CCLCorePlugin (CodeChickenLib-1.10.2-2.2.3.58-universal.jar)
Contact their authors BEFORE contacting forge

// Would you like a cupcake?

Time: 03/10/16 01:39
Description: Rendering entity in world

java.lang.NullPointerException: Rendering entity in world
    at net.minecraft.entity.Entity.func_70068_e(Entity.java:1326)
    at net.minecraft.client.renderer.entity.RenderLivingBase.func_177067_a(RenderLivingBase.java:464)
    at net.minecraft.client.renderer.entity.RenderLivingBase.func_177067_a(RenderLivingBase.java:25)
    at net.minecraft.client.renderer.entity.Render.func_76986_a(SourceFile:54)
    at net.minecraft.client.renderer.entity.RenderLivingBase.func_76986_a(RenderLivingBase.java:204)
    at net.minecraft.client.renderer.entity.RenderPlayer.func_76986_a(RenderPlayer.java:70)
    at net.minecraft.client.renderer.entity.RenderPlayer.func_76986_a(RenderPlayer.java:28)
    at net.minecraft.client.renderer.entity.RenderManager.func_188391_a(RenderManager.java:366)
    at net.torocraft.torohealthmod.gui.GuiEntityStatus.drawEntityOnScreen(GuiEntityStatus.java:166)
    at net.torocraft.torohealthmod.gui.GuiEntityStatus.drawHealthBar(GuiEntityStatus.java:110)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_681_GuiEntityStatus_drawHealthBar_RenderGameOverlayEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:168)
    at net.minecraftforge.client.GuiIngameForge.post(GuiIngameForge.java:888)
    at net.minecraftforge.client.GuiIngameForge.renderExperience(GuiIngameForge.java:586)
    at net.minecraftforge.client.GuiIngameForge.func_175180_a(GuiIngameForge.java:167)
    at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1072)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1076)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
    at net.minecraft.entity.Entity.func_70068_e(Entity.java:1326)
    at net.minecraft.client.renderer.entity.RenderLivingBase.func_177067_a(RenderLivingBase.java:464)
    at net.minecraft.client.renderer.entity.RenderLivingBase.func_177067_a(RenderLivingBase.java:25)
    at net.minecraft.client.renderer.entity.Render.func_76986_a(SourceFile:54)
    at net.minecraft.client.renderer.entity.RenderLivingBase.func_76986_a(RenderLivingBase.java:204)
    at net.minecraft.client.renderer.entity.RenderPlayer.func_76986_a(RenderPlayer.java:70)
    at net.minecraft.client.renderer.entity.RenderPlayer.func_76986_a(RenderPlayer.java:28)

-- Entity being rendered --
Details:
    Entity Type: null (net.minecraft.client.entity.EntityOtherPlayerMP)
    Entity ID: 1578
    Entity Name: mouse961
    Entity's Exact location: -463,73, 75,00, -668,06
    Entity's Block location: World: (-464,75,-669), Chunk: (at 0,4,3 in -29,-42; contains blocks -464,0,-672 to -449,255,-657), Region: (-1,-2; contains chunks -32,-64 to -1,-33, blocks -512,0,-1024 to -1,255,-513)
    Entity's Momentum: -2,40, 0,00, -5,75
    Entity's Passengers: []
    Entity's Vehicle: ~~ERROR~~ NullPointerException: null

-- Renderer details --
Details:
    Assigned renderer: net.minecraft.client.renderer.entity.RenderPlayer@62a33830
    Location: 0,00,0,00,0,00 - World: (0,0,0), Chunk: (at 0,0,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
    Rotation: 0.0
    Delta: 1.0
Stacktrace:
    at net.minecraft.client.renderer.entity.RenderManager.func_188391_a(RenderManager.java:366)
    at net.torocraft.torohealthmod.gui.GuiEntityStatus.drawEntityOnScreen(GuiEntityStatus.java:166)
    at net.torocraft.torohealthmod.gui.GuiEntityStatus.drawHealthBar(GuiEntityStatus.java:110)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_681_GuiEntityStatus_drawHealthBar_RenderGameOverlayEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:168)
    at net.minecraftforge.client.GuiIngameForge.post(GuiIngameForge.java:888)
    at net.minecraftforge.client.GuiIngameForge.renderExperience(GuiIngameForge.java:586)
    at net.minecraftforge.client.GuiIngameForge.func_175180_a(GuiIngameForge.java:167)

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityPlayerSP['LemADEC'/0, l='MpServer', x=-465,45, y=75,88, z=-668,64]]
    Chunk stats: MultiplayerChunkCache: 0, 0
    Level seed: 0
    Level generator: ID 06 - quark_realistic, ver 0. Features enabled: false
    Level generator options: 
    Level spawn location: World: (192,64,256), Chunk: (at 0,4,0 in 12,16; contains blocks 192,0,256 to 207,255,271), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
    Level time: 0 game time, 0 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
    Forced entities: 1 total; [EntityPlayerSP['LemADEC'/0, l='MpServer', x=-465,45, y=75,88, z=-668,64]]
    Retry entities: 0 total; []
    Server brand: BungeeCord (git:BungeeCord-Bootstrap:1.10-SNAPSHOT:1250088:1167) <- fml,forge,sponge
    Server type: Non-integrated multiplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:415)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2660)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:392)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.10.2
    Operating System: Windows 10 (amd64) version 10.0
    Java Version: 1.8.0_92, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 1028238408 bytes (980 MB) / 3125805056 bytes (2981 MB) up to 3125805056 bytes (2981 MB)
    JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms3072m -Xmx3072m
    IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
    FML: MCP 9.32 Powered by Forge 12.18.1.2094 82 mods loaded, 81 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJA  mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
    UCHIJA  FML{8.0.99.99} [Forge Mod Loader] (modpack.jar) 
    UCHIJA  Forge{12.18.1.2094} [Minecraft Forge] (modpack.jar) 
    UCHIJA  backpacks16840{3.1.2} [Backpacks!] (zz-backpacks 1.10.2 - 3.1.2.jar) 
    UCHIJA  slimecompendium{1.4} [Slime Compendium] ([1.10.2]SlimeCompendium1.4.jar) 
    UCHIJA  actuallyadditions{1.10.2-r63} [Actually Additions] (ActuallyAdditions-1.10.2-r63.jar) 
    UCHIJA  mantle{1.10.2-1.0.0.jenkins170} [Mantle] (Mantle-1.10.2-1.0.0.jar) 
    UCHIJA  tconstruct{1.10.2-2.5.3.jenkins387} [Tinkers' Construct] (TConstruct-1.10.2-2.5.3.jar) 
    UCHIJA  immersiveengineering{0.10-42} [Immersive Engineering] (ImmersiveEngineering-0.10-42.jar) 
    UCHIJA  JEI{3.12.2.291} [Just Enough Items] (jei_1.10.2-3.12.2.291.jar) 
    UCHIJA  AppleCore{2.0.1} [AppleCore] (AppleCore-mc1.10.2-2.0.1.jar) 
    UCHIJA  thedragoncore{1.10.2-1.0.2.0} [TheDragonCore] (thedragoncore-1.10.2-1.0.2.0.jar) 
    UCHIJA  armorplus{1.10.2-7.0.0.0} [ArmorPlus] (armorplus-1.10.2-7.0.0.0.jar) 
    UCHIJA  Morphtool{1.0-11} [Morph-o-Tool] (Morph-o-Tool-1.0-11.jar) 
    UCHIJA  Quark{r1.0-56} [Quark] (Quark-r1.0-56.jar) 
    UCHIJA  AutoRegLib{1.0-2} [AutoRegLib] (AutoRegLib-1.0-2.jar) 
    UCHIJA  babymobs{1.5.1} [Baby Mobs] (BabyMobs-1.10.2-1.5.1.jar) 
    UCHIJA  Baubles{1.2.1.0} [Baubles] (Baubles-1.9.4-1.2.1.0.jar) 
    UCHIJA  bdlib{1.12.2.8} [BD Lib] (bdlib-1.12.2.8-mc1.10.2.jar) 
    UCHIJA  betteragriculture{0.16} [Better Agriculture] (BetterAgriculture_(MC-1.10)-0.16.jar) 
    UCHIJA  betterbuilderswands{0.6.5} [Better Builder's Wands] (BetterBuildersWands-0.6.5-1.10r138+8595fc7.jar) 
    UCHIJA  BiomeTweaker{2.1.225} [BiomeTweaker] (BiomeTweaker-1.10.2-2.1.225.jar) 
    UCHIJA  Botania{r1.9-325} [Botania] (Botania r1.9-325.jar) 
    UCHIJA  Chameleon{1.10-2.1.7} [Chameleon] (Chameleon-1.10-2.1.7.jar) 
    UCHIJA  chameleoncreepers{1.4.1} [Chameleon Creepers] (chameleoncreepers-1.4.1-1.10.2.jar) 
    UCHIJA  chisel{MC1.9.4-0.0.6.48} [Chisel] (Chisel-MC1.9.4-0.0.6.48.jar) 
    UCHIJA  mcmultipart{1.2.1} [MCMultiPart] (MCMultiPart-1.2.1-universal.jar) 
    UCHIJA  chiselsandbits{12.1} [Chisels & Bits] (chiselsandbits-12.1.jar) 
    UCHIJA  cookingforblockheads{4.2.15} [Cooking for Blockheads] (CookingForBlockheads_1.10.2-4.2.15.jar) 
    UCHIJA  MineTweaker3{3.0.11} [MineTweaker 3] (CraftTweaker-1.10.2-3.0.11.jar) 
    UCHIJA  dailies{1.10.2-6} [Bailey's Dailies] (dailies-1.10.2-6.jar) 
    UCHIJA  StorageDrawers{1.10.2-3.2.7} [Storage Drawers] (StorageDrawers-1.10.2-3.2.7.jar) 
    UCHIJA  bitdrawers{0.31} [Drawers & Bits] (DrawersBits-1.10.2-0.31.jar) 
    UCHIJA  lootablebodies{2.2.0} [DrCyano's Lootable Bodies] (DrCyanosLootableBodies_1.10.2-2.2.0.jar) 
    UCHIJA  druidry{2016.9.26-1.9.4} [§2Druidry] (druidry(2016.9.26)(1.9.4).jar) 
    UCHIJA  ezstorage{2.2.3} [EZStorage 2] (ezstorage-2.2.3.jar) 
    UCHIJA  FastLeaveDecay{1.1} [Fast Leave Decay] (FastLeaveDecay-MC1.10.2-1.2.2.jar) 
    UCHIJA  forestry{5.2.13.261} [Forestry] (forestry_1.10.2-5.2.13.261.jar) 
    UCHIJA  gemsplus{1.7} [Gems+] (gemsplus-1.7-mc1.10.2.jar) 
    UCHIJA  gendustry{1.6.4.17} [GenDustry] (gendustry-1.6.4.17-mc1.10.2.jar) 
    UCHIJA  harvestfestival{0.5.3} [Harvest Festival] (Harvest-Festival-1.10.2-0.5.3.jar) 
    UCHIJA  InfernalMobs{1.7.1} [Infernal Mobs] (InfernalMobs-1.10.jar) 
    UCHIJA  inventorytweaks{1.61-58-a1fd884} [Inventory Tweaks] (InventoryTweaks-1.61-58.jar) 
    UCHIJA  ironchest{1.10.2-7.0.11.797} [Iron Chest] (ironchest-1.10.2-7.0.11.797.jar) 
    UCHIJA  JEFB{1.0.3} [Just Enough Forestry Bees] (jefb_1.10.2-1.0.3.jar) 
    UCHIJA  journeymap{1.10.2-5.2.4} [JourneyMap] (journeymap-1.10.2-5.2.4-unlimited.jar) 
    UCHIJA  llibrary{1.7.0} [LLibrary] (llibrary-1.7.0-1.10.2.jar) 
    UCHIJA  jurassicraft{2.0.3} [JurassiCraft] (JurassiCraft-2.0.3.jar) 
    UCHIJA  jeresources{0.5.2.57} [Just Enough Resources] (JustEnoughResources-1.10.2-0.5.2.57.jar) 
    UCHIJA  KaishiWeapons{v0.1} [Kaishi's Weapons Pack] (KaishiWeapons-1.9.4.jar) 
    UCHIJA  levels{r4.1.3} [Levels] (Levels-1.10.2-r4.1.3.jar) 
    UCHIJA  modtweaker{2.0.4} [Mod Tweaker] (ModTweaker2-1.10.2-2.0.4.jar) 
    UCHIJA  mousetweaks{2.6.4} [Mouse Tweaks] (MouseTweaks-2.6.4-mc1.10.2.jar) 
    UCHIJA  netherportalfix{1.0} [Nether Portal Fix] (netherportalfix-mc1.9.4-4.0.1.jar) 
    UCHIJA  recipehandler{0.5} [NoMoreRecipeConflict] (NoMoreRecipeConflict-0.5(1.10.2).jar) 
    UCHIJA  harvestcraft{1.9.4-1.10.2f} [Pam's HarvestCraft] (Pam's HarvestCraft 1.9.4-1.10.2f.jar) 
    UCHIJA  shetiphiancore{3.3.1} [ShetiPhian-Core] (shetiphiancore-1.10.0-3.3.1.jar) 
    UCHIJA  platforms{1.2.0} [Platforms] (platforms-1.10.0-1.2.0.jar) 
    UCHIJA  prefab{1.1.1.3} [Prefab] (prefab-1.1.1.3.jar) 
    UCHIJA  cobwebs{1.0.1} [Realistic Cobwebs Mod] (realistic-cobwebs-1.0.1-MC1.10.2.jar) 
    UCHIJA  refinedstorage{1.1.1} [Refined Storage] (refinedstorage-1.1.1.jar) 
    UCHIJA  xreliquary{1.10.2-1.3.3.571} [Reliquary] (Reliquary-1.10.2-1.3.3.571.jar) 
    UCHIJA  ResourceLoader{1.5.1} [Resource Loader] (ResourceLoader-MC1.9.4-1.5.1.jar) 
    UCHIJA  Roguelike{1.5.4} [Roguelike Dungeons] (RoguelikeDungeons-1.10.2-1.5.4.jar) 
    UCHIJA  roughMobs{1.0} [Rough Mobs] (roughMobs1.0.7-1.10.2.jar) 
    UCHIJA  Sampler{1.66} [Sampler] (sampler-1.66.jar) 
    UCHIJA  SpiceOfLife{1.3.6} [The Spice of Life] (SpiceOfLife-mc1.9.4-1.3.6.jar) 
    UCHIJA  squarebuckets{1.01} [Square Buckets] (squarebuckets-1.01_MC-1.10.2-2016-10-02b.jar) 
    UCHIJA  supercraftingframe{1.2.1.87} [Super Crafting Frame] (supercraftingframe-1.10.2-1.2.1.87.jar) 
    UCHIJA  taiga{${version}} [Tinkers Alloying Addon] (taiga-1.10.2-1.1.3.jar) 
    UCHIJA  tellme{0.3.0} [Tell Me] (tellme-1.10.2-0.3.0.jar) 
    UCHIJA  TinkersAddons{1.0.3} [Tinkers' Addons] (Tinkers' Addons-1.10.x-1.0.3.jar) 
    UCHIJA  torohealthmod{1.10.2-1} [ToroHealth Damage Indicators] (torohealth-1.10.2-1.jar) 
    UCHIJA  trashslot{6.3.6} [TrashSlot] (TrashSlot_1.10.2-6.3.6.jar) 
    UCHIJA  Waila{1.7.0} [Waila] (Waila-1.7.0-B3_1.9.4.jar) 
    UCHIJA  WailaHarvestability{1.1.9} [Waila Harvestability] (WailaHarvestability-mc1.10.2-1.1.9.jar) 
    UCHIJA  waterstrainer{1.3.0} [Water Strainer] (WaterStrainer-1.9.4-1.3.0.jar) 
    UCHIJA  icse{1.1.0.0} [I Can See Everything] (Wawla-1.10.2-2.3.0.206.jar) 
    UCHIJA  wawla{2.3.0.202} [What Are We Looking At] (Wawla-1.10.2-2.3.0.206.jar) 
    UCHIJA  worldedit{6.1.4} [WorldEdit] (worldedit-forge-mc1.10.2-6.1.4-dist.jar) 
    UCHIJA  wtfcore{1.10.2_v1.0} [wtfcore] (WTF-Expedition_1.10.2_PreRelease.jar) 
    UD  BiomeTweakerCore{2.1.225} [BiomeTweaker Core] (minecraft.jar) 
    Loaded coremods (and transformers): 
LoadingPlugin (Quark-r1.0-56.jar)
  vazkii.quark.base.asm.ClassTransformer
ShetiPhian-ASM (shetiphiancore-1.10.0-3.3.1.jar)
  shetiphian.core.asm.ClassTransformer
LoadingPlugin (sampler-1.66.jar)

llibrary (llibrary-1.7.0-1.10.2.jar)
  net.ilexiconn.llibrary.server.core.plugin.LLibraryTransformer
  net.ilexiconn.llibrary.server.core.patcher.LLibraryRuntimePatcher
AppleCore (AppleCore-mc1.10.2-2.0.1.jar)
  squeek.applecore.asm.TransformerModuleHandler
HFTransformer (Harvest-Festival-1.10.2-0.5.3.jar)
  joshie.harvest.asm.HFTransformer
LoadingPlugin (ResourceLoader-MC1.9.4-1.5.1.jar)
  lumien.resourceloader.asm.ClassTransformer
FMLPlugin (InventoryTweaks-1.61-58.jar)
  invtweaks.forge.asm.ContainerTransformer
SCFMLCorePlugin (StalkerCreepers-1.10.2.jar)
  atomicstryker.stalkercreepers.common.SCTransformer
BCModPlugin (zz-backpacks 1.10.2 - 3.1.2.jar)
  brad16840.common.asm.ClassTransformer
BiomeTweakerCore (BiomeTweaker-1.10.2-2.1.225.jar)
  me.superckl.biometweaker.core.BiomeTweakerASMTransformer
CCLCorePlugin (CodeChickenLib-1.10.2-2.2.3.58-universal.jar)
  codechicken.lib.asm.ClassHeirachyManager
  codechicken.lib.asm.RenderHookTransformer
    GL info: ' Vendor: 'Intel' Version: '4.4.0 - Build 20.19.15.4474' Renderer: 'Intel(R) HD Graphics 530'
    Pulsar/tconstruct loaded Pulses: 
        - TinkerCommons (Enabled/Forced)
        - TinkerWorld (Enabled/Not Forced)
        - TinkerTools (Enabled/Not Forced)
        - TinkerHarvestTools (Enabled/Forced)
        - TinkerMeleeWeapons (Enabled/Forced)
        - TinkerRangedWeapons (Enabled/Forced)
        - TinkerModifiers (Enabled/Forced)
        - TinkerSmeltery (Enabled/Not Forced)
        - TinkerGadgets (Enabled/Not Forced)
        - TinkerOredict (Enabled/Forced)
        - TinkerIntegration (Enabled/Forced)
        - TinkerFluids (Enabled/Forced)
        - TinkerMaterials (Enabled/Forced)
        - TinkerModelRegister (Enabled/Forced)
        - chiselsandbitsIntegration (Enabled/Not Forced)
        - WailaIntegration (Enabled/Not Forced)

    List of loaded APIs: 
        * actuallyadditionsapi (23) from ActuallyAdditions-1.10.2-r63.jar
        * AppleCoreAPI (2.1.0) from AppleCore-mc1.10.2-2.0.1.jar
        * Baubles|API (1.1.1.0) from Baubles-1.9.4-1.2.1.0.jar
        * BiomeTweakerAPI (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * BiomeTweakerAPI|event (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * BiomeTweakerAPI|script|object (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * BiomeTweakerAPI|script|pack (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * BiomeTweakerAPI|script|wrapper (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * BiomeTweakerAPI|util (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * BotaniaAPI (81) from Botania r1.9-325.jar
        * ChiselAPI (0.0.1) from Chisel-MC1.9.4-0.0.6.48.jar
        * ChiselAPI|Carving (0.0.1) from Chisel-MC1.9.4-0.0.6.48.jar
        * ChiselsAndBitsAPI (11.7.0) from chiselsandbits-12.1.jar
        * CoFHAPI (1.8.9R1.2.0B1) from refinedstorage-1.1.1.jar
        * CoFHAPI|energy (1.8.9R1.2.0B1) from ImmersiveEngineering-0.10-42.jar
        * ForestryAPI|apiculture (5.0.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|arboriculture (4.2.1) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|circuits (3.1.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|climate (2.1.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|core (5.0.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|farming (2.1.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|food (1.1.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|fuels (3.0.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|genetics (4.7.1) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|greenhouse (5.2.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|hives (4.1.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|lepidopterology (1.3.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|mail (3.0.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|multiblock (3.0.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|recipes (5.4.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|storage (5.0.0) from forestry_1.10.2-5.2.13.261.jar
        * ForestryAPI|world (2.1.0) from forestry_1.10.2-5.2.13.261.jar
        * gendustryAPI (2.3.0) from gendustry-1.6.4.17-mc1.10.2.jar
        * ImmersiveEngineering|API (1.0) from ImmersiveEngineering-0.10-42.jar
        * ImmersiveEngineering|ImmersiveFluxAPI (1.0) from ImmersiveEngineering-0.10-42.jar
        * jeresources|API (0.5.2.57) from JustEnoughResources-1.10.2-0.5.2.57.jar
        * journeymap|client-api (1.2) from journeymap-1.10.2-5.2.4-unlimited.jar
        * journeymap|client-api-display (1.2) from journeymap-1.10.2-5.2.4-unlimited.jar
        * journeymap|client-api-event (1.2) from journeymap-1.10.2-5.2.4-unlimited.jar
        * journeymap|client-api-model (1.2) from journeymap-1.10.2-5.2.4-unlimited.jar
        * journeymap|client-api-util (1.2) from journeymap-1.10.2-5.2.4-unlimited.jar
        * JustEnoughItemsAPI (4.8.0) from jei_1.10.2-3.12.2.291.jar
        * StorageDrawersAPI (1.7.10-1.2.0) from refinedstorage-1.1.1.jar
        * StorageDrawersAPI|config (1.7.10-1.2.0) from StorageDrawers-1.10.2-3.2.7.jar
        * StorageDrawersAPI|event (1.7.10-1.2.0) from StorageDrawers-1.10.2-3.2.7.jar
        * StorageDrawersAPI|inventory (1.7.10-1.2.0) from StorageDrawers-1.10.2-3.2.7.jar
        * StorageDrawersAPI|pack (1.7.10-1.2.0) from StorageDrawers-1.10.2-3.2.7.jar
        * StorageDrawersAPI|registry (1.7.10-1.2.0) from StorageDrawers-1.10.2-3.2.7.jar
        * StorageDrawersAPI|render (1.7.10-1.2.0) from refinedstorage-1.1.1.jar
        * StorageDrawersAPI|storage (1.7.10-1.2.0) from StorageDrawers-1.10.2-3.2.7.jar
        * StorageDrawersAPI|storage-attribute (1.7.10-1.2.0) from refinedstorage-1.1.1.jar
        * SuperScript (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * SuperScript|command (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * SuperScript|object (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * SuperScript|util (1.0.225) from BiomeTweaker-1.10.2-2.1.225.jar
        * WailaAPI (1.3) from Waila-1.7.0-B3_1.9.4.jar
    Launched Version: 1.10.2-forge1.10.2-12.18.1.2094
    LWJGL: 2.9.4
    OpenGL: Intel(R) HD Graphics 530 GL version 4.4.0 - Build 20.19.15.4474, Intel
    GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

    Using VBOs: Yes
    Is Modded: Definitely; Client brand changed to 'fml,forge'
    Type: Client (map_client.txt)
    Resource Packs: 
    Current Language: English (US)
    Profiler Position: N/A (disabled)
    CPU: 8x Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
```

**Comment by frodare** [comment]
We were never able to reproduce this issue, is it still occurring?

**Comment by LemADEC** [comment]
Issue was too frequent, we had to remove the mod long time ago

**Comment by zeriley** [comment]
This issue should be fixed now with release of 1.10-2
